test/TestExpectations.json

```diff
@@ -3653,6 +3743,18 @@
     "parameters": ["cdp", "chrome", "new-headless"],
     "expectations": ["FAIL", "PASS"]
   },
+  {
+    "testIdPattern": "[function.spec] Function interpolateFunction should work",
+    "platforms": ["linux"],
```

Comment:
Is this correct, I think this more of a unit test that needs to be moved. So it should pass on all 3 platforms